.github/workflows/ci.yml

```diff
@@ -65,3 +65,82 @@ jobs:
       - name: Upload to codecov
         run: |
           bash <(curl -s https://codecov.io/bash)
+
+  sentry:
+    runs-on: ubuntu-latest
+    timeout-minutes: 25
+    steps:
+      # Checkout codebase
+      - name: Checkout snuba
+        uses: actions/checkout@v2
+
+      # Checkout Sentry and run integration tests against latest snuba
+      - name: Checkout sentry
+        uses: actions/checkout@v2
+        with:
+          repository: getsentry/sentry
+          path: sentry
+
+      - name: Build snuba docker image
```

Comment:
For caching speed ups, I was looking at this article:
https://evilmartians.com/chronicles/build-images-on-github-actions-with-docker-layer-caching

Notice, I have not yet gone through it and tested it.

Reply:
I have a separate PR to address caching https://github.com/getsentry/snuba/pull/2017/